src/card-stack/card-stack.html

```diff
@@ -34,7 +34,7 @@
 
     <div class="stack">
       <template is="dom-repeat" items="[[stack.cards]]">
-        <game-card position$="[[index]]"
+        <game-card draggable="true" ondragstart="_dragStart(event)" position$="[[index]]"
```

Comment:
Ah yes, MB on the merge, but I was under the impression that doing so would only affect me and give me the updated files so I could use the "New Game" button and not have to spam refresh page to test changes to the source I was doing.